browser/src/control/Control.ContextToolbar.ts

```diff
@@ -0,0 +1,173 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * L.Control.NotebookbarWriter - definition of notebookbar content in Writer
+ */
+
+/* global _ _UNO app Point */
+
+class ContextToolbar {
+	static showContextToolbar(pos: Point): void {
+		this.hideContextToolbar();
+		const container = L.DomUtil.createWithId(
+			'div',
+			'context-toolbar',
+			document.body,
+		);
+		L.DomUtil.addClass(container, 'notebookbar horizontal');
+		container.style.left = pos.x + 'px';
+		container.style.top = pos.y + 'px';
+		app.map.uiManager.notebookbar.builder.build(
```

Comment:
As a starting point - good.

But later please use new instance of a builder for this component. I don't want to guess what happens when we switch mode to compact :)

Reply:
Maybe we can just inherit ContextToolbar from Notebookbar class which would share some code.  (it's still JS, but as a first step we can create `browser/src/control/jsdialog/Component.NoteboobarBase.ts` for example which will steal common things from existing `control/Control.Notebookbar.js`, one step closer to TS)